address.go

```diff
@@ -2,26 +2,16 @@ package faker
 
 import (
 	"reflect"
-)
 
-var address Addresser
+	"github.com/bxcodec/faker/v4/options"
+)
 
 // GetAddress returns a new Addresser interface of Address
 func GetAddress() Addresser {
-	mu.Lock()
-	defer mu.Unlock()
-
-	if address == nil {
-		address = &Address{}
-	}
+	address := &Address{}
```

Comment:
if we remove the mutex usage, is this still safe for concurrent usage?

Reply:
Yes, `address` now become a local variable. Every goroutine will get an independent instance.